Project fails to deploy on Vercel

It looks there's an incorrect casing in `index.js` for the `Header` import.

https://github.com/imse-ty/imsety.com/blob/ad1bd07fbd98546bd046123646c6f52e198b8d58/pages/index.js#L3

## Log

```shell
09:10:28	Failed to compile.
09:10:28	ModuleNotFoundError: Module not found: Error: Can't resolve '../components/Header' in '/vercel/path0/pages'
09:10:28	> Build error occurred
09:10:28	Error: > Build failed because of webpack errors
09:10:28	    at /vercel/path0/node_modules/next/dist/build/index.js:17:924
09:10:28	    at processTicksAndRejections (internal/process/task_queues.js:93:5)
09:10:28	    at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584)
09:10:28	error Command failed with exit code 1.
09:10:28	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
09:10:28	Error: Command "yarn run build" exited with 1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,8 @@
 
 ### Fixed
 
-- Incorrect casing in `index.js` for `Header`.
+- Incorrect casing in `index.js` for `Header`. (#5)
+- Incorrect syntax in `tailwind.config.js`. (#7)
 
 ## [0.1.0]
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Fixed
+
+- Incorrect casing in `index.js` for `Header`.
+
 ## [0.1.0]
 
 Initial development.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+## [0.1.1]
+
 ### Fixed
 
 - Incorrect casing in `index.js` for `Header`. (#5)
@@ -11,5 +13,6 @@
 
 Initial development.
 
-[unreleased]: https://github.com/imse-ty/imsety.com/compare/v0.1.0...HEAD
+[unreleased]: https://github.com/imse-ty/imsety.com/compare/v0.1.1...HEAD
+[0.1.1]: https://github.com/imse-ty/imsety.com/compare/v0.1.0...v0.1.1
 [0.1.0]: https://github.com/imse-ty/imsety.com/releases/tag/v0.1.0

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "imsety.com",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "private": true,
   "scripts": {
     "dev": "next dev",

--- a/pages/index.js
+++ b/pages/index.js
@@ -1,6 +1,6 @@
 import Head from 'next/head';
 import Footer from '../components/footer';
-import Header from '../components/Header';
+import Header from '../components/header';
 import Hero from '../components/hero';
 import VideoReelLoop from '../components/video-reel-loop';
 

--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -1,5 +1,5 @@
 module.exports = {
-  purge: [`./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}`],
+  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
   darkMode: 'media', // or 'media' or 'class'
   theme: {
     fontFamily: {

